node/client/src/lib.rs

```diff
@@ -595,8 +594,7 @@ where
 		+ sp_api::Metadata<Block>
 		+ sp_offchain::OffchainWorkerApi<Block>
 		+ sp_session::SessionKeys<Block>
-		+ sp_authority_discovery::AuthorityDiscoveryApi<Block>
-		+ pallet_contracts_rpc::ContractsRuntimeApi<Block, AccountId, Balance, BlockNumber, Hash>,
```

Comment:
Why?

Reply:
@MRamanenkau Could you please check this part? I believe we need to restore it since it may have been related to the contract issue you resolved in version 9.30.
```
pallet_contracts_rpc::ContractsRuntimeApi<Block, AccountId, Balance, BlockNumber, Hash>,
```

Reply:
@shamilkhan pallet_contracts_rpc was removed [here](https://github.com/paritytech/substrate/pull/12358).